src/examples/aggregate/plotly/line-without-rds.ts

```diff
@@ -1 +1,54 @@
-// TODO - sorry this example has not been written yet, be a pal and submit a PR!
+import * as d3 from 'd3-fetch';
+import { DSVRowArray } from 'd3-dsv';
```

Comment:
Is this a new dependency, will it get added to the stackblitz example? Or do we need to update the config?
![Screen Shot 2020-05-15 at 11 18 24 AM](https://user-images.githubusercontent.com/11969737/82066822-e852e500-969d-11ea-993c-81ff3b17fe91.png)

Looks like this also applies to 
`/src/examples/aggregate/amcharts/line-without-rds.ts`
`/src/examples/aggregate/gcharts/line-without-rds.ts`